python/hawkey/hawkeymodule.cpp

```diff
@@ -195,10 +194,10 @@ PYCOMP_MOD_INIT(_hawkey)
     Py_INCREF(&selector_Type);
     PyModule_AddObject(m, "Selector", (PyObject *)&selector_Type);
     /* _hawkey.Repo */
-    if (PyType_Ready(&repo_Type) < 0)
-        return PYCOMP_MOD_ERROR_VAL;
-    Py_INCREF(&repo_Type);
-    PyModule_AddObject(m, "Repo", (PyObject *)&repo_Type);
+//    if (PyType_Ready(&repo_Type) < 0)
+//        return PYCOMP_MOD_ERROR_VAL;
+//    Py_INCREF(&repo_Type);
+//    PyModule_AddObject(m, "Repo", (PyObject *)&repo_Type);
```

Comment:
Ehh, what is this? Why is this commented out instead of adapted or deleted?

Reply:
Thank you for comment. I will remove it.